src/patchelf.cc

```diff
@@ -1901,6 +1938,9 @@ int mainWrapped(int argc, char * * argv)
         else if (arg == "--no-default-lib") {
             noDefaultLib = true;
         }
+        else if (arg == "--add-debug") {
```

Comment:
Given that your described usecase is not that common, can we rename it to:

```suggestion
        else if (arg == "--add-debug-section") {
```

Than it's clear to the reader what this flag does.

Reply:
If I understand correctly ELF, DT_DEBUG is actually a "tag" in .dynamic "section". My proposal would be to change to --add-debug-tag. Does that work for you?

Reply:
Yes.